lib/fs.js

```diff
@@ -17,6 +18,22 @@ const mkdir = (dir, mode, cb) => {
   });
 };
 
+const recursivlyListDirs = dir => {
+  const list = [dir];
+  let nextDir = dir;
+  const root = dir.startsWith('/') ? '/' : '.';
```

Comment:
To make this OS agnostic:
```suggestion
  const root = path.parse(dir).root || '.';
```